bazel/external/p4runtime.BUILD

```diff
@@ -101,3 +101,14 @@ py_grpc_library(
     srcs = [":p4runtime_proto"],
     deps = [":p4runtime_py_proto"],
 )
+
+py_proto_library(
+    name = "p4data_py_proto",
+    deps = [":p4data_proto"],
+)
+
+py_grpc_library(
```

Comment:
This should not be a grpc library. p4data doesn't expose any grpc service. The only grpc service is provided by p4runtime.proto.
```suggestion
py_proto_library(
```